mods/ts/rules/defaults.yaml

```diff
@@ -1013,8 +1013,9 @@
 
 ^Tree:
 	Inherits@1: ^SpriteActor
+	Inherits@2: ^ExistsInWorld
 	Interactable:
-	HiddenUnderShroud:
+	FrozenUnderFog:
```

Comment:
This is important to the feature to work, but noting that this could potentially bring a significant performance regression (but hopefully now much less than when we changed `FrozenUnderFog` &rarr; `HiddenUnderShroud` in the first place).

Reply:
I have a very similar setup in @OpenHV and we don't suffer from noticeable performance degradations.